google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/stub/changestream/ReadChangeStreamRetryTest.java

```diff
@@ -282,10 +285,10 @@ public void errorAfterSingleCloseStreamShouldResumeWithOriginalRequestTest() {
             .expectInitialRequest()
             .respondWith(closeStreamResponse)
             .respondWithStatus(Code.UNAVAILABLE));
-    // Resume the request with the original request.
-    // We don't care about the response here so just do expectRequest.
-    service.expectations.add(RpcExpectation.create().expectInitialRequest());
+    // Request is not resumed. We'll get a CloseStream and then an UNAVAILABLE error.
     List<ChangeStreamRecord> actualResults = getResults();
+    expect.expect(UnavailableException.class);
```

Comment:
Are we also returning OK status in this case? I think it make sense to return OK status since it's already the end of the current stream.

Reply:
Discussed offline. Let's don't worry about "UNAVAILABLE after CloseStream" since the server will return an OK right after sending a CloseStream.